flexget/ui/src/plugins/series/components/series-episode/series-episode.component.js

```diff
@@ -1,105 +1,79 @@
 (function () {
-  'use strict';
+    'use strict';
 
-  angular
+    angular
     .module('flexget.plugins.series')
     .component('seriesEpisode', {
-      templateUrl: 'plugins/series/components/series-episode/series-episode.tmpl.html',
-      controllerAs: 'vm',
-      controller: seriesEpisodeController,
-      bindings: {
-        episode: '<',
-        show: '<',
-        deleteEpisode: '&',
-        resetReleases: '&'
-      },
+        templateUrl: 'plugins/series/components/series-episode/series-episode.tmpl.html',
+        controllerAs: 'vm',
+        controller: seriesEpisodeController,
+        bindings: {
+            episode: '<',
+            show: '<',
+            deleteEpisode: '&',
+            resetReleases: '&'
+        },
     });
 
     function seriesEpisodeController($mdDialog, $http, $stateParams, $filter){
-      var vm = this;
+        var vm = this;
 
-      //Small hacks to make the accordion work, if that ever gets in Angular Material, this can be replaced
-      var releasesOpen = false;
-      vm.toggleReleases = function() {
-        releasesOpen = !releasesOpen;
+        loadReleases();
```

Comment:
Isn't the button greyed out because of the call from the episode list? I think it has an attribute episode_number_of_releases which it uses, not sure though